docs/api/guilds.rst

```diff
@@ -0,0 +1,281 @@
+.. SPDX-License-Identifier: MIT
+
+.. currentmodule:: disnake
+
+Guilds
+======
+
+This section documents everything related to Guilds - collections of users, roles, channels etc. and main
+hubs for communication in Discord, referred to as a Servers in the official UI.
```

Comment:
link to https://discord.com/developers/docs/resources/guild

Reply:
Done in [`d15ddd7`](https://github.com/DisnakeDev/disnake/pull/722/commits/d15ddd70e20c611070a3445379763a80b07a8c13)